docker/gnn-benchmarking/apply.py

```diff
@@ -11,13 +11,19 @@
 
 from graphnet.deployment.i3modules import I3InferenceModule
 from graphnet.data.extractors.i3featureextractor import (
-    I3FeatureExtractorIceCubeDeepCore,
+    I3FeatureExtractorIceCubeUpgrade,
 )
 from graphnet.data.constants import FEATURES
+from graphnet.constants import GRAPHNET_ROOT_DIR
 
 
-# Constants (from Dockerfile)
-MODEL_PATH = "model.pth"
+# Constants
+MODEL_NAME = "total_neutrino_energy"
+BASE_PATH = (
+    f"{GRAPHNET_ROOT_DIR}/src/graphnet/models/pretrained/icecube/upgrade/QUESO"
+)
+MODEL_CONFIG = f"{BASE_PATH}/{MODEL_NAME}/{MODEL_NAME}_config.yml"
+STATE_DICT = f"{BASE_PATH}/{MODEL_NAME}/{MODEL_NAME}_state_dict.yml"
```

Comment:
I believe you used the wrong file extension for the state_dict.

Reply:
Thanks. The dangers of copy paste...